libkbfs/bserver_remote.go

```diff
@@ -253,23 +252,15 @@ func (b *blockServerRemoteClientHandler) OnDisconnected(ctx context.Context,
 
 // ShouldRetry implements the ConnectionHandler interface.
 func (b *blockServerRemoteClientHandler) ShouldRetry(rpcName string, err error) bool {
-	//do not let connection.go's DoCommand retry any batch rpcs automatically
-	//because i will manually retry them without successfully completed references
+	// Do not let connection.go's DoCommand retry any batch rpcs
+	// since batchDowngradeReferences already handles retries.
 	switch rpcName {
 	case "keybase.1.block.delReferenceWithCount":
 		return false
 	case "keybase.1.block.archiveReferenceWithCount":
 		return false
-	case "keybase.1.block.archiveReference":
```

Comment:
We don't call archiveReference anymore, and it doesn't call batchDowngrade anyway.